packages/react/src/components/text-area/text-area.tsx

```diff
@@ -159,11 +160,17 @@ export const TextArea: VoidFunctionComponent<TextAreaProps> = ({
                 <Counter
                     data-testid="char-counter"
                     aria-live="polite"
-                    aria-hidden="true"
                     id={idCounter}
                     valid={inputValueLength <= maxLength}
                 >
-                    {t('characters', { current: inputValueLength, max: maxLength })}
+                    {inputValueLength}
```

Comment:
@LarryMatte I was thinking if this way would be the same and better?

```
<span aria-live="polite" aria-label="12 of 30 characters available">
  12 / 30 characters
</span>
```

Reply:
BTW I just had this though immediately after doing my PR

Reply:
That would improve code readability quite a bit!